drivers/bluetooth/hci/spi.c

```diff
@@ -36,6 +36,7 @@ LOG_MODULE_REGISTER(bt_driver);
 /* Offsets */
 #define STATUS_HEADER_READY	0
 #define STATUS_HEADER_TOREAD	3
+#define STATUS_HEADER_TOWRITE	1
```

Comment:
Where did this come from? Offset 1 in the HCI controller sample is `SANITY_CHECK`, nothing to do with writing.